### PR TITLE
Adding relative indexing method

### PR DESCRIPTION
**Pull request by Rumyra**
Adding at() method

### CODE REVIEW

File: files/en-us/mozilla/firefox/experimental_features/index.html [hunk 1]
```diff
@@ -513,6 +513,42 @@ <h3 id="Property_math-style">Property: math-style </h3>
 
 <h2 id="JavaScript">JavaScript</h2>
 
+<h3 id="Relative_indexing_method">Relative indexing method</h3>
+
+<p>The relative indexing method <code>at()</code> has been added to the Array, String and TypedArray global objects, allowing relative indexing from the end when passed a negative index. See <a href="/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at">Array.prototype.at()</a>, <a href="/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/at">String.prototype.at()</a> and <a href="/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/at">TypedArray.prototype.at()</a> respectively.</p>
```
Comment (chrisdavidmills) [inline files/en-us/mozilla/firefox/experimental_features/index.html:518]: * make `Array`, `String` and `TypedArray` into links to their respective global object landing pages. Make sure they are wrapped in `<code>` elements also.

* I think you need to expand the description of what this does and make it clear that you can use it to return items from index positions relative to the start AND end. "allowing relative indexing from the end when passed a negative index" kind of implies that it only does it from the end. 

* Wrap the links to the specific methods (e.g. `Array.prototype.at()`) in `<code>` elements too.

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Mostly looking good, but I've got a few comments for you.

Cheers @Rumyra !

**Review by chrisdavidmills — APPROVED** [review]
I've made a couple of teaks, but this looks good now!

Thanks for being awesome @Rumyra

File: files/en-us/mozilla/firefox/experimental_features/index.html [hunk 2]
```diff
@@ -515,7 +515,7 @@ <h2 id="JavaScript">JavaScript</h2>
...
+<p>The relative indexing method <code>at()</code> has been added to the <code><a href="/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array">Array</a></code>, <code><a href="/en-US/docs/Web/JavaScript/Reference/Global_Objects/String">String</a></code> and <code><a href="/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray">TypedArray</a></code> global objects. Passing an integer to the method returns the item or character at that position. It allows for negative integers which count back from the end of the array or string. For example `1` would return the second item or character and `-1` would return the last item or character. See <code><a href="/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at">Array.prototype.at()</a></code>, <code><a href="/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/at">String.prototype.at()</a></code> and <code><a href="/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/at">TypedArray.prototype.at()</a></code> respectively.</p>
```
Comment (Ryuno-Ki) [inline files/en-us/mozilla/firefox/experimental_features/index.html]: Does it make sense to wrap `<a>` inside `<code>` instead of the other way round?

Comment (Rumyra) [reply files/en-us/mozilla/firefox/experimental_features/index.html]: I've kept it consistent with how the macros do it

**Pull request merged**